README.md

```diff
@@ -1,54 +1,133 @@
-xblock-group-project-v2
-=======================
+# xblock-group-project-v2
+
 
 This XBLock is experimental reimplementation of [Group Project XBlock](https://github.com/edx-solutions/xblock-group-project).
 
 It does *not* work properly in LMS yet.
 
-Configuration variables
------------------------
+## Features
 
-This block uses following configuration variables in the ``XBLOCK_SETTINGS``:
+**Hierarchical structure**
+ 
+Group projects are organized around activities and stages: 
 
-    "XBLOCK_SETTINGS": {
-      "group_project_v2": {
-        "dashboard_details_url": "/admin/workgroup/dashboard/programs/{program_id}/courses/{course_id}/projects/{project_id}/details?activate_block_id={activity_id}",
-        "ta_review_url": "/courses/{course_id}/group_work/{group_id}?activate_block_id={activate_block_id}",
-        "access_dashboard_for_all_orgs_groups": ["super_admin"],
-        "access_dashboard_groups": ["client_admin"],
-        "ta_roles": ["assistant"]
-      }
-    },
+    * Activities are the larger chunks, used to distinguish between different parts of group project. Activities 
+      can contain one or more stages. Grading happens at activity level, i.e. each student receive one grade per 
+      activity.
+    * Stages are smaller building blocks, representing individual steps towards project completion.
```

Comment:
@e-kolpakov Could you remove the leading spaces from these items to avoid making them render as code? The same comment applies to the items under "Grading" and "Omnipresent features".